scripts/generate_docs.sh

```diff
@@ -108,6 +108,7 @@ if [[ $PUSH == true ]]; then
   rm -rf "docs/$version"
   rm -rf "docs/current"
   cp -r "$jazzy_dir/docs/$version" docs/
+  cp -r "$jazzy_dir/docset/$version" docset/
```

Comment:
iirc the better fix would be to remove the `--docset-path` argument. I originally added it so that the docset won't be included in what we push to gh-pages. the reason was size - this is quite big and make the repo fairly heavy afaict.

Reply:
Makes sense - worked for me, and should be pretty straightforward to do. I'll update the PR tomorrow for that.

Reply:
@tomerd size considerations are real. I think @gwynne was already saying that NIO takes a very long time to check out?

Should we maybe start moving the generated docs to a different repository?

Reply:
yes, this is a think I wanted to suggest. we can potentially also post docs to s3 and server them up from there.

Reply:
ping @weissi wdyt?

Reply:
@tomerd I'm 100% in favour of S3 or separate repo. I feel you're much better positioned to choose what the best option is. If we can keep the generated docs outside this repo, I'd be happier :)